apps/evm/lib/evm/builtin.ex

```diff
@@ -7,6 +7,8 @@ defmodule EVM.Builtin do
   TODO: Implement and add doc tests.
   """
 
+  @g_identity 15 + 3
```

Comment:
Could we split this into two module variables? I think that would make it more clear that 15 is the base price and it costs 3 gas per byte